### ISSUE TITLE
Subspace Inference for Bayesian Deep Learning

### ISSUE THREAD

**Issue by juanmc2005** [juanmc2005/phd-bib#7]
## [Subspace Inference for Bayesian Deep Learning](https://arxiv.org/abs/1907.07504)
*Bayesian inference was once a gold standard for learning with neural networks, providing accurate full predictive distributions and well calibrated uncertainty. However, scaling Bayesian inference techniques to deep neural networks is challenging due to the high dimensionality of the parameter space. In this paper, we construct low-dimensional subspaces of parameter space, such as the first principal components of the stochastic gradient descent (SGD) trajectory, which contain diverse sets of high performing models. In these subspaces, we are able to apply elliptical slice sampling and variational inference, which struggle in the full parameter space. We show that Bayesian model averaging over the induced posterior in these subspaces produces accurate predictions and well calibrated predictive uncertainty for both regression and image classification.*

### Cite this paper
```bibtex


@article{subspace_bayesian2019,
  author    = {Pavel Izmailov and
               Wesley J. Maddox and
               Polina Kirichenko and
               Timur Garipov and
               Dmitry P. Vetrov and
               Andrew Gordon Wilson},
  title     = {Subspace Inference for Bayesian Deep Learning},
  journal   = {CoRR},
  volume    = {abs/1907.07504},
  year      = {2019},
  url       = {http://arxiv.org/abs/1907.07504},
  archivePrefix = {arXiv},
  eprint    = {1907.07504},
  timestamp = {Tue, 23 Jul 2019 10:54:22 +0200},
  biburl    = {https://dblp.org/rec/bib/journals/corr/abs-1907-07504},
  bibsource = {dblp computer science bibliography, https://dblp.org}
}
```
